### PR TITLE
https://issues.redhat.com/browse/ACM-10773 4/10/24 ACM 2.10.1 z-strea…

### PR DESCRIPTION
**Pull request by cadawson**
…m docs errata

### CODE REVIEW

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -7,5 +7,14 @@ By default, Errata updates are automatically applied when released. The details
 
 See link:../install/upgrade_hub.adoc#upgrading-by-using-the-operator[Upgrading by using the operator] for more information about upgrades.
 
-No {product-title} errata updates for this version at this time.
+== Errata 2.10.1
+
+* Fixes a problem that might occur for users who use the {product-title} backup and recovery function and backed up the `managedcluster` namespace without using the _cluster.open-cluster-management.io/backup: cluster-activation_ label.  The problem caused the managed cluster namespace to remain in the `Terminating` state after it is restored. (link:https://issues.redhat.com/browse/ACM-9780[ACM-9780])
```
Comment (dockerymick) [inline release_notes/errata.adoc]: Are you intending to add emphasis to the label ? I would moonospace 
```suggestion
* Fixes a problem that might occur for users who use the {product-title} backup and recovery function and backed up the `managedcluster` namespace without using the `cluster.open-cluster-management.io/backup: cluster-activation` label.  The problem caused the managed cluster namespace to remain in the `Terminating` state after it is restored. (link:https://issues.redhat.com/browse/ACM-9780[ACM-9780])
```

File: release_notes/errata.adoc [hunk 2]
```diff
@@ -7,5 +7,14 @@ By default, Errata updates are automatically applied when released. The details
...
+
+* Fixes the issue when a policy might temporarily be set to `noncompliant` with a message of `context cancelled` when the policy was being updated while the `governance-policy-framework` pod was shutting down on the managed cluster. (link:https://issues.redhat.com/browse/ACM-10402[ACM-10402])
```
Comment (dockerymick) [inline release_notes/errata.adoc]:
```suggestion
* Fixes the issue where a policy might temporarily be set to `noncompliant` with a message of `context cancelled` when the policy was being updated, while the `governance-policy-framework` pod was shutting down on the managed cluster. (link:https://issues.redhat.com/browse/ACM-10402[ACM-10402])
```

**Review by dockerymick — APPROVED** [review]
Peer review complete

**Comment by cadawson** [comment]
Thanks @dockerymick !

**Review by rhaignere — APPROVED** [review]
LGTM, thank you!
Thanks @cadawson and @dockerymick!

**Comment by dockerymick** [comment]
Received approvals, but the i see that the advisory is not live yet. holding on this PR

File: release_notes/errata.adoc [hunk 3]
```diff
@@ -7,5 +7,14 @@ By default, Errata updates are automatically applied when released. The details
 
 See link:../install/upgrade_hub.adoc#upgrading-by-using-the-operator[Upgrading by using the operator] for more information about upgrades.
 
-No {product-title} errata updates for this version at this time.
+== Errata 2.10.1
+
+* Fixes a problem that might occur for users who use the {product-title} backup and recovery function and backed up the `managedcluster` namespace without using the `cluster.open-cluster-management.io/backup: cluster-activation` label.  The problem caused the managed cluster namespace to remain in the `Terminating` state after it is restored. (link:https://issues.redhat.com/browse/ACM-9780[ACM-9780])
```
Comment (dockerymick) [inline release_notes/errata.adoc]:
```suggestion
* Fixes a problem that might occur for users who use the {product-title} backup and recovery function and backed up the `managedcluster` namespace without using the `cluster.open-cluster-management.io/backup: cluster-activation` label. The problem caused the managed cluster namespace to remain in the `Terminating` state after it is restored. (link:https://issues.redhat.com/browse/ACM-9780[ACM-9780])
```

**Review by dockerymick — APPROVED** [review]

**Pull request merged**